### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created cart page containing order details
index 1
index-2

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
Code looks good, just make sure to remove the extra black spaces.

File: stylesheets/cart.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.cart-horz-card .badge-rating {
+    max-width: fit-content;
+    background-color: var(--green-500);
+    padding: 4px 8px;
+}
+
+.cart-product-info {
+    margin-top: 1.5rem;
+    width: 15rem;
+
```
Comment (rahulcs754) [inline stylesheets/cart.css]: remove space

**Review by rahulcs754 — APPROVED** [review]
code is good. try to prettier to remove space in CSS and HTML file .

File: stylesheets/cart.css (hunk 1 above)
Comment (nakulsharma15) [reply stylesheets/cart.css]: just did in the latest commit. Thanks for the suggestion

**Pull request merged**